core/src/main/scala/commandcenter/command/StocksCommand.scala

```diff
@@ -0,0 +1,70 @@
+package commandcenter.command
+
+import com.typesafe.config.Config
+import commandcenter.CCRuntime.Env
+import commandcenter.command.StocksCommand.StocksResult
+import commandcenter.view.DefaultView
+import io.circe.{ Decoder, Json }
+import sttp.client.circe.asJson
+import sttp.client.httpclient.zio.SttpClient
+import sttp.client.{ basicRequest, UriContext }
+import zio.{ IO, TaskManaged, ZIO, ZManaged }
+
+final case class StocksCommand(commandNames: List[String], tickers: List[Ticker]) extends Command[Unit] {
+  val commandType: CommandType = CommandType.StocksCommand
+  val title: String            = "Stock"
+  val stocksUrl: String        = "https://query1.finance.yahoo.com/v7/finance/quote?symbols="
+
+  def preview(searchInput: SearchInput): ZIO[Env, CommandError, List[PreviewResult[Unit]]] =
+    for {
+      input    <- ZIO.fromOption(searchInput.asPrefixed).orElseFail(CommandError.NotApplicable)
+      symbols   = if (input.rest.isBlank) tickers.map(_.id).mkString(",") else input.rest
+      request   = basicRequest
+                    .get(uri"$stocksUrl$symbols")
+                    .response(asJson[Json])
+      response <- SttpClient
+                    .send(request)
+                    .map(_.body)
+                    .absolve
+                    .mapError(CommandError.UnexpectedException)
+      stocks   <- IO.fromEither(
+                    response.hcursor.downField("quoteResponse").downField("result").as[List[StocksResult]]
+                  ).mapError(CommandError.UnexpectedException)
+    } yield stocks.map { stock =>
+      Preview.unit
+        .score(Scores.high(input.context))
+        .view(
+          DefaultView(
+            title,
```

Comment:
I think you can get rid of the `Stocks` title since it's self-explanatory from the output.

You can pass the `fansi.Str` directly into `view()` since there's a View instance for it already defined.